index.html

```diff
@@ -1,208 +1,340 @@
-        <div class="specifics">
-          <h3><span>&#9744;</span>Frameworks</h3>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSpec</li>
-            <li>CapyBara</li>
-            <li>Salenium</li>
-          </ul>
-        </div>
-        <div class="specifics">
-          <h3><span>&#9679;</span>Skills</h3>
-          <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </div>
-      </div>
-    </section>
-    <script src="script.js"></script>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./dist/style.css" />
+  </head>
+  <body>
+    <header class="navigation">
+      <div class="logo">
+        <h2><a href="">Evans Karanja</a></h2>
+      </div>
+      <nav class="navlinks">
+        <ul>
+          <li class="navItem" onclick="closeMenu()">
+            <a href="#portfolio">Portfolio</a>
+          </li>
+          <li class="navItem" onclick="closeMenu()">
+            <a href="#about">About</a>
+          </li>
+          <li class="navItem" onclick="closeMenu()">
+            <a href="#contact">Contact</a>
+          </li>
+          <li class="navItem icn">
+            <a href="#email"><i class="fa-solid fa-envelope"></i></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+    <section class="intro">
+      <div class="intro-text">
+        <h1>Hey There. <br />I'm Evans</h1>
+        <h2>I'm a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="socials">
+        <ul>
+          <li>
+            <a href=""><i class="fa-brands fa-github"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-solid fa-m"></i></a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section id="portfolio">
+      <div class="title">
+        <h2>My Recent Works</h2>
+        <hr />
+      </div>
+      <div class="plchdr">
+        <div class="img">
+          <img src="/images/plc-img.jpg" alt="" />

```

Comment:
[OPTIONAL] please fix the image path for your projects also, as it is not displayed I think it should be `src="./images/plc-img.jpg"`